lua/modules/avatar.lua

```diff
@@ -325,12 +325,18 @@ index.getEyesColor = function(_, playerOrHead)
 end
 
 index.setBodyPartColor = function(_, _, shape, skin1, skin2)
-	if skin1 ~= nil and shape.Palette[SKIN_1_PALETTE_INDEX] then
-		shape.Palette[SKIN_1_PALETTE_INDEX].Color = skin1
+	if type(skin1) ~= "Color" then
+		error("avatar:setBodyPartColor: skin1 must be a Color", 2)
```

Comment:
It should be possible to only set skin1 or skin2, that's why it was designed like this.
Now it would show an error, can you change this please?